python-sdk/src/astro/utils/workflow_directory_parser.py

```diff
@@ -10,11 +10,53 @@
 from airflow.models import BaseOperator
 from yaml.scanner import ScannerError
 
+from astro.constants import GENERATED_WORKFLOW_INCLUDE_DIRECTORY
+from astro.custom_backend.serializer import deserialize
 from astro.sql import LoadFileOperator
 from astro.sql.operators.transform import TransformOperator
-from sql_cli.constants import GENERATED_WORKFLOW_INCLUDE_DIRECTORY
-from sql_cli.operators.load_file import get_load_file_instance
```

Comment:
It made sense to me to keep operator related methods separate (we may not want to call them operators, I agree), because with more operators coming in , we might populate this module with more operator specific builder methods.

Reply:
@pankajkoti ultimately we need to keep all of this stuff in the python SDK if we want render_file to work without the SQL CLI. If you would like me to put it somewhere else please suggest an alternative location.

Reply:
I think we can keep it like in this PR and move it to alternative location if needed in upcoming implementations.